### PR TITLE
AFK - InsaneCoder789

### PR DESCRIPTION
**Pull request by InsaneCoder789**
Signed-off-by: 
InsaneCoder789

I have Changed the AFK command
Check and Have a Look 👍 :UwU:

### CODE REVIEW

**Comment by MysteryCoder456** [comment]
Can you summarize what you've changed in the PR?

**Comment by InsaneCoder789** [comment]
Everything ! 
Gave it a Database so you dont have to use `UnAFK` anymore 
Also Gave it a embed

**Comment by InsaneCoder789** [comment]
also its Benedict's AFK 
And you know that how it works

**Comment by MysteryCoder456** [comment]
> Everything ! 
> 
> Gave it a Database so you dont have to use `UnAFK` anymore 
> 
> Also Gave it a embed

I'm pretty sure it already used a database. From what I've seen at a glance you're just storing the afks in a dictionary.

**Comment by InsaneCoder789** [comment]
yes I am

**Comment by MysteryCoder456** [comment]
> yes I am 

Can you maybe revert all the changes where you removed the Sqlalchemy stuff and changed it to use a dictionary? I'll take a better look at your other changes when I'm available.

**Comment by InsaneCoder789** [comment]
Ok , But its a PR and it doesnt affect the main file unless you dont pull it
- That is atleast what i know about how Github Works

**Comment by MysteryCoder456** [comment]
> Ok , But its a fucking PR and it doesnt affect the main file unless you dont pull it
> 
> - That is atleast what i know about how Github Works 


I know it's a PR that's not merged. I'll check your changes when I can. Meanwhile please try to be more civil. This conversation is public.

**Comment by MysteryCoder456** [comment]
Ok so I've looked at the code and I haven't found anything useful, unfortunately. You've only removed the SQLAlchemy code in favor of using a dictionary to store the AFKs, which in my opinion is a downgrade because all the AFKs will be lost when the bot restarts. So I'll be closing this PR because it only seems to reduce the bot's functionality.
